pkg/operator/ceph/cluster/crash/reconcile.go

```diff
@@ -175,6 +177,12 @@ func (r *ReconcileNode) reconcile(request reconcile.Request) (reconcile.Result,
 			}
 		}
 
+		secret := &corev1.Secret{}
+		err = r.client.Get(context.TODO(), types.NamespacedName{Name: crashCollectorKeyName, Namespace: namespace}, secret)
+		if err != nil {
+			return reconcile.Result{Requeue: true, RequeueAfter: 5 * time.Second}, errors.Wrapf(err, "could not list the %q secret.", crashCollectorKeyName)
```

Comment:
Since this is only during initial creation and the time to initialize the mons is more in the range of 1-3 minutes, how about a longer requeue?  
```suggestion
			return reconcile.Result{Requeue: true, RequeueAfter: 30 * time.Second}, errors.Wrapf(err, "could not list the %q secret.", crashCollectorKeyName)
```

Reply:
done.